front/README.md

```diff
@@ -84,35 +84,104 @@ Afterwards:
 
 ## Testing
 
-Testing is done with React Testing Library and Jest.
+Testing is done with [Playwright](https://playwright.dev/) for application, integration, and component testing.
 
-Test files are located in the same directory as the files they are testing. For example, `EditBox.test.js` and `EditBox.tsx` are both located in `front/src/views/EditBox`.
+[Vitest](https://vitest.dev/) is used for standalone TypeScript functions and modules.
 
-For integration tests, we mock the Apollo client with a `MockedProvider` component instead of the `ApolloProvider` component that is used to handle real data. More information on mocking the Apollo client can be found [here](https://www.apollographql.com/docs/react/development-testing/testing/).
+We use [Mock Service Worker](https://mswjs.io/) (MSW for short) to mock data in a way that faithfully mimics the real server. It's also very useful to mock GraphQL Schema bits that aren't implemented in the production backend yet to build new features on top of it.
 
-To eliminate repetitive code, a custom renderer was built in `front/src/tests/test-utils.js`. It allows developers to render a component in a test environment where chakra, Apollo and Routes are wrapped around it. The utility also exports the entire react testing library, so you should import from this utility instead of `@testing-library/react`. See `EditBox.test.js` for examples of the custom renderer's use.
+Application test specs are inside the `/tests` folder. Other tests are colocated with the files that are the subject of what is being tested.
 
-Tests and test coverage can be run with the following command:
+See Playwright's docs to see how to make the most of its API to help you build test cases e.g. [recording tests by using the app](https://playwright.dev/docs/codegen-intro).
 
```

Comment:
please add two steps how to install it locally. To me pnpm install was not enough. It also asked me to install the browsers. Since downloading all browsers were quite large I only installed chromium with:

`pnpm exec playwright install chromium --with-deps`